big-int/src/operators/less_than.cpp

```diff
@@ -32,5 +32,47 @@
 
 namespace libbig
 {
+  bool largeInt:: operator < (const largeInt& z) {
+    // sign = 1: positive number
+    // sign = 0: negative number
+    // flip if its reverse
 
+    bool NEGATIVE = 0;
```

Comment:
it is good to move these `NEGATIVE` and `POSITIVE` to `bigint.hpp` as `constexpr bool`'s because they will be used in several other files, so we can define them in header file itself